### PR TITLE
BMR-26: Top app bar - MAKU

### PR DESCRIPTION
**Pull request by ma-za-kpe**
https://github.com/AmaliTech-Training-Rw/Meeting-Room-booking-App/assets/28810111/1df534f3-1490-467c-92cb-a6d16c6525a2

### CODE REVIEW

**Comment by ma-za-kpe** [comment]
Hey @esthcarelle , the colors were changed by someone back to only white mode. That's why we are using default colors.

File: core_ui/src/main/java/com/amalitech/core_ui/components/BookMeetingTopAppBar.kt [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import com.amalitech.core_ui.state.BookMeetingRoomAppState
+import com.amalitech.core_ui.state.rememberBookMeetingRoomAppState
+import com.amalitech.core_ui.theme.BookMeetingRoomTheme
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun BookMeetingTopAppBar(
+    appState: BookMeetingRoomAppState,
```
Comment (esthcarelle) [inline core_ui/src/main/java/com/amalitech/core_ui/components/BookMeetingTopAppBar.kt:30]: The UI does not match the figma UI, could you please check it out?

**Review by esthcarelle — CHANGES_REQUESTED** [review]

File: core_ui/src/main/java/com/amalitech/core_ui/components/BookMeetingTopAppBar.kt (hunk 1 above)
Comment (ma-za-kpe) [reply core_ui/src/main/java/com/amalitech/core_ui/components/BookMeetingTopAppBar.kt:33]: - I've updated the size of the image to 32.dp
- font-weight, and size updates too!
- added a shadow, (ill need your feedback especially regarding this one)
est

Comment (ma-za-kpe) [reply core_ui/src/main/java/com/amalitech/core_ui/components/BookMeetingTopAppBar.kt:33]: @esthcarelle , PTAL

**Review by bensek — APPROVED** [review]

**Review by esthcarelle — APPROVED** [review]

**Pull request merged**